packages/blazor-workspace/NimbleBlazor/package.json

```diff
@@ -26,6 +26,6 @@
   ],
   "peerDependencies": {
     "cross-env": "^7.0.3",
```

Comment:
Got past the npm install issues via #2321 but now there are build issues. I haven't looked into them yet.
```
Error: ../../node_modules/prosemirror-markdown/dist/index.d.ts(3,19): 
error TS7016: Could not find a declaration file for module 
'markdown-it/lib/token'. '/home/runner/work/nimble/nimble/node_modules/markdown-it/lib/token.js' 
implicitly has an 'any' type.
```

Reply:
Traced it down I think. 92% confident the issue is prosemirror-markdown builds against an older version of [`@types/markdown-it`](https://github.com/ProseMirror/prosemirror-markdown/blob/e59521463f62b296cc072962aa2d60b267ea4397/package.json#L32) and doesn't know that this [deep import of the Token](https://github.com/ProseMirror/prosemirror-markdown/blob/e59521463f62b296cc072962aa2d60b267ea4397/src/from_markdown.ts#L3) from markdown-it is not how the `@types/markdown-it` package models the [v14 exports](https://github.com/surenpoghosian/DefinitelyTyped/blob/315e71956e855af59a77682d7b45d75aa10539d0/types/markdown-it/package.json#L13-L16) compared to [v13](https://github.com/surenpoghosian/DefinitelyTyped/blob/315e71956e855af59a77682d7b45d75aa10539d0/types/markdown-it/v13/package.json) which didn't define explicit exports.

I think (85% confidence) the issue is that the `./*` export config means that resolving `import Token from 'markdown-it/lib/token';` fails because that does not resolve to an actual export. Need to either:
- rely on the barrel file `import Token from 'markdown-it';`
- use an [explicit file](https://unpkg.com/browse/markdown-it@14.1.0/lib/) that would match, in this case `import Token from 'markdown-it/lib/token.mjs';`

The [node docs](https://nodejs.org/api/packages.html#package-entry-points) cover it:

> To make the introduction of ["exports"](https://nodejs.org/api/packages.html#exports) non-breaking, ensure that every previously supported entry point is exported. It is best to explicitly specify entry points so that the package's public API is well-defined. For example, a project that previously exported main, lib, feature, and the package.json could use the following package.exports:

```
{
  "name": "my-package",
  "exports": {
    ".": "./lib/index.js",
    "./lib": "./lib/index.js",
    "./lib/index": "./lib/index.js",
    "./lib/index.js": "./lib/index.js",
    "./feature": "./feature/index.js",
    "./feature/index": "./feature/index.js",
    "./feature/index.js": "./feature/index.js",
    "./package.json": "./package.json"
  }
} 
```

Reply:
Created an issue in prosemirror-markdown for their usage of markdown-it exports: https://github.com/ProseMirror/prosemirror-markdown/issues/124

It's a workaround, but we can opt rennovate out of updating this package for now.

Reply:
@jattasNI making a separate PR with this and the [rollup change](https://github.com/ni/nimble/pull/2318#discussion_r1700858311) below

Reply:
Ended up not doing the rollup change as discussed in the other thread, but disabling this update in https://github.com/ni/nimble/pull/2327
Resolving and hoping this PR is good to go after.